exchanges/websocket/wshandler/wshandler_types.go

```diff
@@ -204,3 +168,42 @@ type WebsocketConnection struct {
 	ResponseMaxLimit     time.Duration
 	TrafficTimeout       time.Duration
 }
+
+// ProtocolFeatures holds all variables for the exchanges supported features
+// for a protocol
+type ProtocolFeatures struct {
```

Comment:
This introduces many copies of these big protocol structs. Would be nice to export features to its own package under exchanges so we can rid this, plus won't experience any import cycles similar to how we do it with the asset package. Also has the added benefit of not having to update them all individually.

Reply:
Yeah, I wasn't happy with this one, but had import cycle problems. Didn't like the 3rd duplication, however golang is strict when it comes to applying it. If someone ever changed anything on one struct but not the other, it won't compile, so it wouldn't crate too much work to fix.
I'll look into moving it to its own area that doesn't create an import problem 👍